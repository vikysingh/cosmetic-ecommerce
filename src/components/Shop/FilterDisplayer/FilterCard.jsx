import React from "react"
import { CloseCircleOutlined } from "@ant-design/icons"

import styles from "./FilterDisplayer.module.css"

export default function FilterCard({ type, name }) {
    return <div className={styles.FILTER_DISPLAYER__filter_card} >
        <label>{type}: </label>
        <strong> { name } </strong>
        <CloseCircleOutlined id={styles.FILTER_DISPLAYER__close_icon} />
    </div>
}