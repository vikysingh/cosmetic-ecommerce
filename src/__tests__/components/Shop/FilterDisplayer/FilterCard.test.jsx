import React from "react"
import { shallow } from "enzyme"

import FilterCard from "../../../../components/Shop/FilterDisplayer/FilterCard"
import { dummyFilterData } from "../../../../test_files/dummyTestData"

const { type,name, dispatch, action } = dummyFilterData

test('Should render FilterCard from Shop/FilterDisplayer', ()=> {
    const wrapper = shallow(<FilterCard type={type} name={name} dispatch={dispatch} action={action} />)

    expect(wrapper).toMatchSnapshot()
})