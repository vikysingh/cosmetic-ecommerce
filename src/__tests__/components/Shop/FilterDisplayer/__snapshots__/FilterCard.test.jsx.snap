// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render FilterCard from Shop/FilterDisplayer 1`] = `
<div
  className="FILTER_DISPLAYER__filter_card"
>
  <label>
    brand
    : 
  </label>
  <strong>
     
    Maybelline
     
  </strong>
  <[object Object]
    id="FILTER_DISPLAYER__close_icon"
    onClick={[Function]}
  />
</div>
`;
